native/vendor/google/billing/GoogleBillingHelper.cpp

```diff
@@ -42,555 +48,354 @@ namespace {
 }; // namespace
 
 namespace cc {
-namespace {
-std::string callStringMethod(JNIEnv* env, jclass clazz, jobject obj, const char* methodName) {
-    jmethodID methodId = env->GetMethodID(clazz, methodName, "()Ljava/lang/String;");
-    jobject jStringObj = env->CallObjectMethod(obj, methodId);
-    if (jStringObj != nullptr) {
-        return cc::StringUtils::getStringUTFCharsJNI(env, static_cast<jstring>(jStringObj));
-    }
-    return "";
-}
-jobject callObjectMethod(JNIEnv* env, jclass clazz, jobject obj, const char* methodName, const char* returnType) {
-    std::string returnSign = cc::StringUtils::format("()L%s", returnType);
-    jmethodID methodId = env->GetMethodID(clazz, methodName, returnSign.c_str());
-    return env->CallObjectMethod(obj, methodId);
-}
-
-int callIntMethod(JNIEnv* env, jclass clazz, jobject obj, const char* methodName) {
-    jmethodID methodId = env->GetMethodID(clazz, methodName, "()I");
-    return env->CallIntMethod(obj, methodId);
-}
 
-bool callBooleanMethod(JNIEnv* env, jclass clazz, jobject obj, const char* methodName) {
-    jmethodID methodId = env->GetMethodID(clazz, methodName, "()Z");
-    return env->CallBooleanMethod(obj, methodId);
-}
+void invokeJsCallback(se::Object* obj, const char* jsFunctionName, const se::ValueArray& args) {
```

Comment:
Put `invokeJsCallback` to an anonymous namespace.

Reply:
fixed.